services/QuillLMS/spec/services/pdf_file_builder_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe PdfFileBuilder do
+  subject { described_class.run(data, template) }
+
+  let(:data) { { some_key: 'some_value' } }
+  let(:template) { 'pdf' }
+  let(:pdf_file_builder) { described_class.new(data, template) }
```

Comment:
Is this `pdf_file_builder` in use in this file, or is it a leftover from a previous set of tests that stubbed things differently?

Reply:
Whoops.  Yes, I originally tested some of the private methods but ended up scrapping them.  This was a leftover.